src/utils/validation-utils/attributes-validation/triangles-calculation.ts

```diff
@@ -2,14 +2,14 @@ import type { MeshAttribute } from '@loaders.gl/schema';
 
 import { Vector3 } from "@math.gl/core";
 
-const scratchVector = new Vector3();
-
 /**
  * Calculate triangle vertices of tile
  * @param attribute
  * @param offset
  */
 export const getTriangleVertices = (attribute: MeshAttribute, offset: number): Vector3[] => {
+  const scratchVector = new Vector3();
```

Comment:
scratchVector should be initialized as global variable.

Reply:
Done